### DISCUSSION TITLE
Consider migrating to Codeberg

### DISCUSSION THREAD

**Discussion by ThousandDemonDaggers** [Floorp-Projects/Floorp#1319] [Feature Request]
Moving everything to [Codeberg](https://codeberg.org) would be great, as it is a libre, free, open-source, privacy-friendly and non-profit git platform from Germany, running the Gittea soft-fork [Forgejo](https://forgejo.org).
Instead, GitHub is closed source software owned by Microsoft, opposing to Floorp.

* https://codeberg.org/Recommendations/Mirror_to_Codeberg
* https://nogithub.codeberg.page

**Comment by regularentropy** [comment]
> Moving everything to [Codeberg](https://codeberg.org) would be great, as it is a libre, free, open-source, privacy-friendly and non-profit git platform from Germany, running the Gittea soft-fork [Forgejo](https://forgejo.org). Instead, GitHub is closed source software owned by Microsoft, opposing to Floorp.
> 
> * https://codeberg.org/Recommendations/Mirror_to_Codeberg
> * https://nogithub.codeberg.page

Thank you for the suggestion, but I don't see the reason should we move there. 
Apart from being “libre, foss, privacy-friendly” and this sort of stuff, what advantages does it practically offer to the development process?

**Comment by ThousandDemonDaggers** [comment]
Aside from having more control over the project (self-hosting) and reflecting a FLOSS ideology more closely, no, there is no real practical advantage as far as I know.
